schemas/v0.2/projjson.schema.json

```diff
@@ -0,0 +1,981 @@
+{
+  "$id": "https://proj.org/schemas/v0.2/projjson.schema.json",
```

Comment:
`docs/source/usage/projjson.rst` needs to be updated to version 0.2. I it not immediately clear to me if there's version 0.2 features that needs to be highlighted in the docs.

Reply:
added in a new commit, with a history section